cli/flox-rust-sdk/src/providers/git.rs

```diff
@@ -638,6 +641,28 @@ impl From<GitCommandError> for GitRemoteCommandError {
                 debug!("Ref not found: {ref_name}");
                 GitRemoteCommandError::RefNotFound(ref_name.to_string())
             },
+            GitCommandError::BadExit(_, _, ref stderr)
+                if stderr.starts_with(REMOTE_BRANCH_NOT_FOUND_ERR_PREFIX) =>
+            {
+                let branch_name = stderr
+                    .strip_prefix(REMOTE_BRANCH_NOT_FOUND_ERR_PREFIX)
+                    .unwrap();
```

Comment:
Unwraps are likely to cause panics so you probably need to handle the error case

Reply:
Would `unwrap_or` suffice?

Reply:
Yes since that provides a default value in case there's an error. A bare `unwrap` causes a panic.

Reply:
This is in an if branch that checks for the prefix beforehand and we use unwraps in the other branches as well for that reason